components/Feature/Services/index.js

```diff
@@ -40,6 +46,20 @@ const Services = ({
     }
   };
 
+  const sendFeedback = () => {
+    let enteredText = document.getElementById('search-feedback').value;
```

Comment:
Hm I'd probably put that text box and send button into a form and then get the value of entered text from event.target. 
I think using forms for things like this also works better for screen readers

Reply:
Added a form and it picks it up from target now.